app/views/admin_posts/show.html.erb

```diff
@@ -25,7 +25,8 @@
       <h4 class="heading">
         Published: <%= @admin_post.created_at %>  
         <% if logged_in_as_admin? %>
-            <%= link_to ts("Edit Post"), edit_admin_post_path(@admin_post), :class => 'action' %>
+            <%= link_to ts("Edit Post"), edit_admin_post_path(@admin_post), class: 'action' %>
```

Comment:
`class: action` can be removed here as well, and the comments I left on the other file apply here as well for line 29.